### ISSUE TITLE
Support -A/-B options in grep

### ISSUE THREAD

**Issue by turbanoff** [tools4j/unix4j#56]
grep has options `-A`/`-B` which useful to get context of every matched line.

       -A NUM, --after-context=NUM
              Print NUM lines of trailing context after matching lines.  Places a line containing a group separator (described under --group-separator) between contiguous groups of matches.  With the -o or --only-matching  option,  this  has  no  effect  and  a
              warning is given.

       -B NUM, --before-context=NUM
              Print  NUM  lines  of  leading  context  before  matching lines.  Places a line containing a group separator (described under --group-separator) between contiguous groups of matches.  With the -o or --only-matching option, this has no effect and a
              warning is given.

It would be nice to have support of this options in unix4j

**Comment by terzerm** [comment]
You are right those options would be a good addition.  Just not sure I'll have time to work on this in the next few weeks.
